catalog/task/collect-data/0.2/collect-data.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: collect-data
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to collect extra data from release resources and optionally save CRs to a workspace
+  params:
+    - name: release
+      type: string
+      description: The namespaced name of the Release
+    - name: releaseplan
+      type: string
+      description: The namespaced name of the ReleasePlan
+    - name: releaseplanadmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+    - name: releasestrategy
+      type: string
+      description: The namespaced name of the ReleaseStrategy
+    - name: snapshot
+      type: string
+      description: The namespaced name of the Snapshot
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  results:
+    - name: fbcFragment
+      description: The first component's containerImage in the snapshot to use in the fbc pipelines
+  steps:
+    - name: collect-data
+      image: quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31
+      env:
+        - name: "RELEASE"
+          value: '$(params.release)'
+        - name: "RELEASE_PLAN"
+          value: '$(params.releaseplan)'
+        - name: "RELEASE_PLAN_ADMISSION"
+          value: '$(params.releaseplanadmission)'
+        - name: "RELEASE_STRATEGY"
+          value: '$(params.releasestrategy)'
+        - name: "SNAPSHOT"
+          value: '$(params.snapshot)'
+        - name: "SUBDIR"
+          value: '$(workspaces.data.path)/$(params.subdirectory)'
+      script: |
+        #!/usr/bin/env sh
+        set -e
+
+        if [ -n "$(params.subdirectory)" ]; then
+          mkdir -p $SUBDIR
```

Comment:
could probably just `cd` in after this and not have to write it on every line after, but fine either way

Reply:
Good idea, done.